src/aui/auibook.cpp

```diff
@@ -2712,7 +2712,7 @@ void wxAuiNotebook::OnTabDragMotion(wxAuiNotebookEvent& evt)
                 {
                     wxRect hint_rect = tab_ctrl->GetClientRect();
                     tab_ctrl->ClientToScreen(&hint_rect.x, &hint_rect.y);
-                    m_mgr.ShowHint(hint_rect);
+                    m_mgr.UpdateHint(tab_ctrl->GetRect());
                     return;
```

Comment:
I don't understand this change:
- `hint_rect` is calculated but not used!
- We pass a rectangle (in client coordinates) to `UpdateHint()`, but it accepts a rectangle in screen coordinates? or, the intention here is to use `tab_ctrl->GetScreenRect()` instead?

Reply:
Oops, thanks for noticing this. This is just a bad rebase conflict resolution, it should be `UpdateHint(hint_rect)`, of course.

I thought about changing the code to use `GetScreenRect()` too, but decided it wasn't worth doing it, but maybe it is...